src/main/java/net/nullspace_mc/tapestry/mixin/feature/fillupdates/SetBlockCommandMixin.java

```diff
@@ -24,6 +24,11 @@ public abstract class SetBlockCommandMixin extends Command {
     )
     private boolean disableUpdate(World world, int x, int y, int z, Block block, int metadata, int flags) {
         SetBlockHelper.applyFillUpdatesRule = true;
-        return world.setBlockWithMetadata(x, y, z, block, metadata, Settings.fillUpdates ? 3 : 2);
+
+        if (!Settings.fillUpdates) {
+            flags &= ~SetBlockFlags.UPDATE_NEIGHBORS;
```

Comment:
This uses non-intuitive at first glance operators, can we have a comment ? something along the lines of 

```
flags &= ~SetBlockFlags.UPDATE_NEIGHBORS; // turn off update neighbors flag
```

Reply:
done